interfaces/Portal/src/environments/environment.dev.ts

```diff
@@ -0,0 +1,25 @@
+// THIS FILE IS GENERATED BY 'npm run set-env-variables'
```

Comment:
This file should've been ignored by git, I think... So it shouldn't be committed.

Reply:
In gitignore I only see those:

# Build Configuration/secrets:
.env
environment.prod.ts

Should I also add: environment.dev.ts?

Reply:
@elwinschmitz I have done the changes based on our conversations. Could you please check it now. The PR becomes small now :)